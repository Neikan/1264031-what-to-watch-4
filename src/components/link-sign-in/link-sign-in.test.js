// Импорт библиотек
import React from "react";
import renderer from "react-test-renderer";
import {Router} from "react-router-dom";
import history from "../../history.js";

// Импорт компонентов
import LinkSignIn from "./link-sign-in.jsx";


describe(`Test LinkSignIn component`, () => {
  test(`LinkSignIn component is created and rendered correctly`, () => {
    const tree = renderer.create(
        <Router history={history}>
          <LinkSignIn />
        </Router>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
