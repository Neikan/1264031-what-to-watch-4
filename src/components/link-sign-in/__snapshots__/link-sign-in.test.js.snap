// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test LinkSignIn component LinkSignIn component is created and rendered correctly 1`] = `
<a
  className="user-block__link"
  href="/login"
  onClick={[Function]}
>
  Sign in
</a>
`;
